cmd/api/.air.toml

```diff
@@ -0,0 +1,49 @@
+# Config file for [Air](https://github.com/cosmtrek/air) in TOML format
+
+# Working directory
+# . or absolute path, please note that the directories following must be under root.
+root = "."
+tmp_dir = "tmp"
```

Comment:
I think we can move this `tmp` into this folder `cmd/api`.
Because in case that we have another `tmp`, it will break some parts.